tornettools/generate_defaults.py

```diff
@@ -30,6 +27,8 @@
 TOR_DIR_PORT=8080
 TOR_GUARD_MIN_CONSBW=2000
 
+PROC_ENV="OPENSSL_ia32cap=~0x200000200000000;EVENT_NOSELECT=1;EVENT_NOPOLL=1;EVENT_NOKQUEUE=1;EVENT_NODEVPOLL=1;EVENT_NOEVPORT=1;EVENT_NOWIN32=1"
```

Comment:
I can't remember, but did we decide that Tor actually doesn't need any of these env variables to be set in order to work correctly? If so, should we simplify and remove them?

Reply:
I added them only because they were there originally. The `OPENSSL_ia32cap=~0x200000200000000` to disable hardware AES might be useful once we have a preload that skips encryption.

Reply:
I think we decided [here](https://github.com/shadow/shadow/issues/1370) that preloading the AES cipher does not actually improve performance, so it's easier to just not deal with the extra complexity of another preload library. Without a preload, I think we would want to enable hardware AES to make sure it's as efficient as possible, right?

Reply:
I forgot about that. I've removed these environment variables.